GliaWidgets/Public/Glia/Glia.swift

```diff
@@ -383,30 +409,25 @@ extension Glia {
             switch event {
             case .screenShareOffer(answer: let answer):
                 self?.environment.coreSdk.requestEngagedOperator { operators, _ in
-                    self?.callVisualizer.offerScreenShare(
-                        from: operators ?? [],
-                        configuration: Theme().alertConfiguration.screenShareOffer,
+                    self?.operatorRequestHandlerService.offerScreenShare(
+                        from: operators?.compactMap { $0.name }.joined() ?? "",
```

Comment:
would this make joined string like `JohnJoeCarlMari`? perhaps we should use some separator there?

Reply:
I was using the same logic that was present before. I will look into improving it.

Reply:
fixed [2bff344](https://github.com/salemove/ios-sdk-widgets/pull/885/commits/2bff344a639d14c0048afecba3007635c01d515e)